apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/models/Sgv2Table.java

```diff
@@ -71,129 +45,61 @@ public TableOptions getTableOptions() {
+      return clusteringKey.contains(key);
     }
   }
 
   // copy of SGv1 TableOptions
   @JsonInclude(JsonInclude.Include.NON_NULL)
-  public static class TableOptions {
-    private Integer defaultTimeToLive;
-    private List<ClusteringExpression> clusteringExpression;
-
-    @JsonCreator
-    public TableOptions(
-        @JsonProperty("defaultTimeToLive") final Integer defaultTimeToLive,
-        @JsonProperty("clusteringExpression")
-            final List<ClusteringExpression> clusteringExpression) {
-      this.defaultTimeToLive = defaultTimeToLive;
-      this.clusteringExpression = clusteringExpression;
-    }
-
+  public record TableOptions(
+      @Schema(
+              description =
+                  "Defines the Time To Live (TTL), which determines the time period (in seconds) to expire data. If the value is >0, TTL is enabled for the entire table and an expiration timestamp is added to each column. The maximum value is 630720000 (20 years). A new TTL timestamp is calculated each time the data is updated and the row is removed after the data expires.")
+          Integer defaultTimeToLive,
+      @Schema(
+              description =
+                  "Order rows storage to make use of the on-disk sorting of columns. Specifying order can make query results more efficient. Defaults to ascending if not provided.")
+          List<ClusteringExpression> clusteringExpression) {
     public TableOptions() {
       this(null, null);
     }
-
-    @Schema(
-        description =
-            "Defines the Time To Live (TTL), which determines the time period (in seconds) to expire data. If the value is >0, TTL is enabled for the entire table and an expiration timestamp is added to each column. The maximum value is 630720000 (20 years). A new TTL timestamp is calculated each time the data is updated and the row is removed after the data expires.")
-    public Integer getDefaultTimeToLive() {
-      return defaultTimeToLive;
-    }
-
-    @Schema(
-        description =
-            "Order rows storage to make use of the on-disk sorting of columns. Specifying order can make query results more efficient. Defaults to ascending if not provided.")
-    public List<ClusteringExpression> getClusteringExpression() {
-      return clusteringExpression;
-    }
-
-    public void setDefaultTimeToLive(int defaultTimeToLive) {
-      this.defaultTimeToLive = defaultTimeToLive;
-    }
-
-    public void setClusteringExpression(List<ClusteringExpression> clusteringExpression) {
-      this.clusteringExpression = clusteringExpression;
-    }
   }
 
   // copied from SGv1 ClusteringExpression
-  public static class ClusteringExpression {
+  public record ClusteringExpression(
+      @Schema(required = true, description = "The name of the column to order by") String column,
+      // 18-Jul-2022, tatu: Originally with DropWizard, Swagger 3, we had:
+      //    allowableValues = "ASC,DESC"
+      // but hoping "enumeration" works as replacement
```

Comment:
does it work? And what's the difference from `allowableValues`?

Reply:
I think it does based on comments I saw back in the day: will remove the comment -- new property is the replacement for the old one.